spells.yml

```diff
@@ -6453,11 +6863,9 @@ mirror_image:
   components: V, S
   concentration: false
   description: |-
-    Three illusory duplicates of yourself appear in your space. Until the spell ends, the duplicates move with you and mimic your actions, shifting position so it's impossible to track which image is real. You can use a standard action to dismiss the illusory duplicates.
-
-    Each time a creature targets you with an attack during the spell's duration, roll a d20 to determine whether the attack instead targets one of your duplicates.
+    Three illusory duplicates of yourself appear in your space. Until the spell ends, the duplicates move with you and mimic your actions, shifting position so it's impossible to track which image is real. You can use a standard Magic action to dismiss the illusory duplicates.
 
-    If you have three duplicates, you must roll a 6 or higher to change the attack's target to a duplicate. With two duplicates, you must roll an 8 or higher. With one duplicate, you must roll an 11 or higher.
+    Each time a creature targets you with an attack during the spell's duration, randomly determine whether the attack instead targets one of your duplicates. You and each duplicate have equal chances of being hit. For example, if you have three duplicates, there is a 25% chance the attack targets you and a 75% chance that it targets one of your duplicates.
```

Comment:
I'm not sure I'm a fan of outright putting percentages here. It feels antithetical to D&D to not frame it in terms of dice, even if this does make more sense mathematically.